beets/util/__init__.py

```diff
@@ -94,6 +96,14 @@ def log(self, logger):
             logger.debug(self.tb)
         logger.error(u'{0}: {1}', self.error_kind, self.args[0])
 
+    def log_as_warn(self, logger):
+        """Log to the provided `logger` a human-readable message as a
+        warning and a verbose traceback as a debug message.
+        """
+        if self.tb:
+            logger.debug(self.tb)
+        logger.warn(u'{0}: {1}', self.warn_kind, self.args[0])
+
```

Comment:
I totally get the motivation here, but I also think it would be fine to log these as errors—the operation really did encounter a failure to do what the user asked to do, which I think rises above the level of a warning.